src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlSerializer.cs

```diff
@@ -468,18 +469,21 @@ protected virtual SamlAuthenticationStatement ReadAuthenticationStatement(XmlDic
                 if (string.IsNullOrEmpty(authInstance))
                     throw LogReadException(LogMessages.IDX11115, SamlConstants.Elements.AuthenticationStatement, SamlConstants.Attributes.AuthenticationInstant);
 
-                var authenticationInstant = DateTime.ParseExact(
+                authenticationStatement.AuthenticationInstant = DateTime.ParseExact(
```

Comment:
Please modify LogMessages.IDX11115, IDX11116, IDX11117, IDX11119 . Change: 'Unable to read for SamlSecurityToken' to: 'Unable to read SamlSecurityToken'.  Remove the word 'for'